.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: node_js
 node_js:
-  - 0.10
+- 0.1
 before_install:
-  - "npm i -g bower karma-cli"
+- npm i -g bower karma-cli
```

Comment:
Removal of spaces is not correct yaml (although it may work)

Reply:
in this case the "-" is perceived as part of the indentation, so it will work. But I don't think it improves on readability. http://yaml.org/spec/1.2/spec.html#id2777534

Reply:
I used the travis-cli to add the upload to the npm it might have changed the formatting (i didn't removed them manually). I don't have any preference but i think it's better to go along with the travis cli on this

Reply:
Sure, no problem.